### ISSUE TITLE
Cockpit shadow moves with headtracking

### ISSUE THREAD

**Issue by morallo** [Prof-Butts/xwa_ddraw_d3d11#86]
I realized that the cockpit shadows move slightly when you move your head in VR mode.
This should not happen, as long as the ship does not move, the shadows should be static with respect to the cockpit model.

It happens both with raytracing and shadow mapping modes.

Any ideas why this could be @Prof-Butts ?

**Comment by Prof-Butts** [comment]
Yeah, I think the shadow map is probably computed at the wrong time. It should be computed after the head tracking has been applied, but maybe it's done before that (?) so that's why there's a small delay. Maybe the fix is to apply the "head transform delta" when computing the shadow map (?)

**Comment by Prof-Butts** [comment]
I fixed this a while ago. The only pending issue here is the shadow displacement in external view.
